packages/lib/src/interactions/utils.ts

```diff
@@ -99,3 +99,21 @@ export function clampRectangleToVis(
   const shift = newCenter.clone().sub(center);
   return [startPoint.clone().add(shift), endPoint.clone().add(shift)];
 }
+
+export function isModifierKey(key: unknown): key is ModifierKey {
+  return typeof key === 'string' && ['Alt', 'Control', 'Shift'].includes(key);
+}
+
+export function getModifierKeyArray(
+  keys: ModifierKey | ModifierKey[] | undefined
+): ModifierKey[] {
+  if (!keys) {
+    return [];
+  }
+
+  if (isModifierKey(keys)) {
+    return [keys];
+  }
```

Comment:
Not sure about this -- might be cleaner to just use `Array.isArray(keys)` and leave `isModifierKey` as it was before (with a `key: string` argument, if I recall).

```ts
export function getModifierKeyArray(
  keys: ModifierKey | ModifierKey[] | undefined = []
): ModifierKey[] {
  if (Array.isArray(keys)) {
    return keys;
  }

  return [keys];
}
```

Which can be considerably shortened with a ternary, which makes me think that it might not be worth extracting into a function:

```ts
const modifierKeys = Array.isArray(modifierKey) ? modifierKey : [modifierKey];
```